### ISSUE TITLE
[Rated Disabilities] Move app folder / Isolate application

### ISSUE THREAD

**Issue by jerekshoe** [department-of-veterans-affairs/va.gov-team#66061]
## Background
The rated-disabilities folder is currently nested under `src/applications/personalization`. The `personalization` folder is primarily used by the authenticated experience team for their applications and it makes it easier to deal with CODEOWNERS if we move our apps out of the `personalization` folder. This will make it easier for us to isolate the application and enable continuous deployment as well

### Related documentation
https://depo-platform-documentation.scrollhelp.site/developer-docs/isolated-application-builds

### Considerations
* We have a slack user group called `@benefits-app-builds` set up already. We use it with the `claims-status` application which is already isolated

### Add entry to the allow-list
* Relevant file in vets-website: `config/changed-apps-build.json`
* Code to add:
```json
{
    "rootFolder": "rated-disabilities",
    "slackGroup": "@benefits-app-builds"
}
```

## Tasks
- [x] Move the rated-disabilities application out of the personalization folder
- [ ] Update CODEOWNERS
```[tasklist]
### Isolate rated-disabilities application
- [x] Run check-app-imports script
- [x] Run megaMenu script
- [x] Add entry to the allow-list and create a PR
```

## Acceptance Criteria
- [ ] The Rated disabilities app is in it's own folder
- [ ] The Rated disabilities app is configured for continuous deployment

**Comment by jerekshoe** [comment]
**Update:** It turns out that there were a few places where things were being imported from the rated disabilities application into other apps. I haven't checked to see if the opposite is also happening, but I merged updates to deal with the imports from other apps this morning

**Comment by jerekshoe** [comment]
**Update:** I made a copy of the `rated-disabilities` folder and put it in `src/applications` directory. After talking to other FE devs, I am realizing that the best path forwards is to keep the old folder and create a new entry in the `content-build` repo with a new path. Once that is in place and we have confirmed that both version of the app (the original and the copy) are working. We can swap the paths in `content-build`. Once we confirm those are working as well, we can remove the extra entry and remove the original app folder. Once this process is done, we can begin the process of setting the app up as an isolated app

**Comment by jerekshoe** [comment]
**Update:** I'm going to go ahead and run the scripts that need to be ran to verify that the application is ready to be added to the allow-list and will post the output of those scripts below. We should be able to do this part in parallel to the `content-build` changes I mentioned above

**Comment by jerekshoe** [comment]
**Command:**
```
yarn check-app-imports --app-folders rated-disabilities
```

**Script output:**
```sh
yarn run v1.19.1
$ node script/check-cross-app-imports.js --app-folders rated-disabilities
Analyzing app imports...
No cross app imports were found!
✨  Done in 209.81s.
```

**Comment by jerekshoe** [comment]
### MegaMenu check
**Command:**
```
# app_url value comes from the rootUrl property in manifest.json
yarn cy:run --spec src/platform/site-wide/mega-menu/tests/megaMenu.cypress.spec.js --env app_url=disability/view-disability-rating/my-rating
```

**Script output (some lines removed for brevity):**
```
  (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  megaMenu.cypress.spec.js                  82ms        5        1        -        4        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                         82ms        5        1        -        4        -

✨  Done in 6.85s.
```

**Comment by jerekshoe** [comment]
**Update:** PR is out to add the app to the allow-list. I'm realizing that the other `vets-website` PR needs to be merged before the one for `content-build` can be merged, because the `content-build` build checks won't pass until the new folder has been deployed to dev. Once I get the first `vets-website` PR merged, I can work on merging the second one, as well as the first `content-build` PR

**Comment by jerekshoe** [comment]
**Update (10/23):**
* Merged the [first PR](https://github.com/department-of-veterans-affairs/vets-website/pull/26240) for `vets-website` to duplicate the app folder to `src/applications/rated-disabilities`.
* Going to try to get the [PR to add the new application to the isolated app build allow-list](https://github.com/department-of-veterans-affairs/vets-website/pull/26300) merged tomorrow (10/24).
* The dev deployment seems to be having issues currently and I have to wait for dev to be deployed before I can get the PR for the `content-build` repo merged. 
* Once the above are done, I can verify that the new app is working at its' temporary URL before making a new PR to `content-build` to swap the URLs

**Comment by jerekshoe** [comment]
**Update (10/25):**
* [PR to add the new application to the isolated app build allow-list](https://github.com/department-of-veterans-affairs/vets-website/pull/26300) has been merged
* The PR for the `content-build` repo has been merged
* Put a new PR for `content-build` to switch the PRs in draft
  * Need to merge a PR to remove the `va-breadcrumbs` web component from the new rated-disabilities application because I'm using the content-build ones now instead
* Created a new PR for `vets-website` to swap the `rootUrls` and e2e tests

**Comment by jerekshoe** [comment]
**Update (10/31):**
* Last PR for content-build is ready (will gather approvals / merge tomorrow after sprint review)
* Last PR for vets-website is ready (waiting on content-build one to be merged so that build checks will pass)

Once the 2 PRs above are merged, this ticket will be ready to close
